message.go

```diff
@@ -29,6 +29,8 @@ type MessageOutgoing struct {
 	Metadata Metadata
 	// Payload is the message's Payload.
 	Payload json.RawMessage
+	// Delay (seconds) is the time when the message should become visible in the queue.
+	Delay int
```

Comment:
the `time.Duration` could be a bit nicer type.

We using it here:
https://github.com/dataddo/pgq/blob/main/consumer.go#L334
https://github.com/dataddo/pgq/blob/main/consumer.go#L479

https://pkg.go.dev/github.com/jackc/pgtype#Interval

TBH, no idea how it performs in comparison to `make_interval`

Reply:
Hi, maybe yes, I've chosen int for simplicity/[KISS](https://en.wikipedia.org/wiki/KISS_principle). In publisher.go It's necesarry add interval to time.Now() where with time.Duration query will be more complex and I think second is enough small interval. As I mentioned in my first post I think using SQL server time is better to have "same time source".